charts/updatetree/values.yaml

```diff
@@ -0,0 +1,23 @@
+namespace:
+  name: trillian-system
+serviceAccount:
+  name: trillian-logserver
+  create: false
```

Comment:
can you add `annotations`, so it would show up in the README.md and schema.

thanks

Reply:
done!